e2e/rdctl.e2e.spec.ts

```diff
@@ -163,9 +164,15 @@ test.describe('Command server', () => {
     const rawSettings = resp.body.read().toString();
 
     resp = await doRequest('/v0/settings', rawSettings, 'PUT');
-    expect(resp.ok).toBeTruthy();
-    expect(resp.status).toEqual(202);
-    expect(resp.body.read().toString()).toContain('no changes necessary');
+    expect({
```

Comment:
So we generally like these parallel tests, but what happens if `resp.ok` is false and you throw an exception while evaluating `resp.body.read()` ? Is this a possibility?

Reply:
I believe that on failure (e.g., 404), there is still a response body to read from.  On something lower level (like connection refused), `fetch` throws an exception and we never get here.

So yeah, using `resp.body.read()` is fine here.

(Tested by changing the path to get a 404, and hacking `doRequest` to use an invalid hostname for the second case.)